src/main/java/de/chojo/repbot/listener/SelfCleanupService.java

```diff
@@ -0,0 +1,13 @@
+package de.chojo.repbot.listener;
+
+import net.dv8tion.jda.api.sharding.ShardManager;
+
+public class SelfCleanupService implements Runnable {
+
+    ShardManager shardManager;
+
+    @Override
+    public void run() {
+        shardManager.getGuilds();
+    }
+}
```

Comment:
What is this about? From my view, it does nothing useful

Reply:
It doesnt belong there, but I was to lazy to remove it again xD